public/app/plugins/panel/graph3/LineStyleEditor.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useMemo } from 'react';
+import { FieldOverrideEditorProps, SelectableValue } from '@grafana/data';
+import { HorizontalGroup, IconButton, LineStyle, RadioButtonGroup, Select } from '@grafana/ui';
+
+type lineFill = 'solid' | 'dash' | 'dot';
+
+const LINE_OPTIONS: Array<SelectableValue<lineFill>> = [
+  {
+    label: 'Solid',
+    value: 'solid',
+  },
+  {
+    label: 'Dash',
+    value: 'dash',
+  },
+  // {
+  //   label: 'Dots',
+  //   value: 'dot',
+  // },
+];
+
+const SPACKING_DASH: Array<SelectableValue<string>> = [
+  '10, 10', // default
+  '20, 5',
+  '20, 5',
+  '15, 3, 3, 3',
+  '20, 3, 3, 3, 3, 3, 3, 3',
+  '12, 3, 3',
+].map(txt => ({
```

Comment:
Is there any logic to these?  :) seems like it would be better to provide more of the standard ones (dash length + space) 

These last 3 just look very strange to me, are these patterns useful common?